zeebe/backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/AzureBackupStore.java

```diff
@@ -92,6 +92,28 @@ public static BlobServiceClient buildClient(final AzureBackupConfig config) {
     }
   }
 
+  BlobContainerClient getContainerClient(
+      final BlobServiceClient client, final AzureBackupConfig config) {
+    final BlobContainerClient blobContainerClient =
+        client.getBlobContainerClient(config.containerName());
+
+    if (!config.createContainer()) {
+      LOG.info(
```

Comment:
:upside_down_face: Should be debug IMO

```suggestion
      LOG.debug(
```